cfme/tests/infrastructure/test_vm_reconfigure.py

```diff
@@ -196,34 +196,64 @@ def test_vm_reconfig_add_remove_hw_hot(provider, small_vm, ensure_vm_running, ch
     reconfigure_vm(small_vm, orig_config)
 
 
-@pytest.mark.manual
 @pytest.mark.tier(1)
-@pytest.mark.parametrize('change_type', ['sockets', 'memory'])
-def test_vm_reconfig_add_remove_hw_hot_vmware(change_type):
+@pytest.mark.parametrize('disk_type', ['thin', 'thick'])
+@pytest.mark.parametrize(
+    'disk_mode', ['persistent', 'independent_persistent', 'independent_nonpersistent'])
+@pytest.mark.provider([VMwareProvider], override=True)
+def test_vm_reconfig_add_remove_disk_hot(small_vm, disk_type, disk_mode):
+
     """
     Polarion:
         assignee: nansari
         initialEstimate: 1/6h
         testtype: functional
-        startsin: 5.5
+        startsin: 5.9
         casecomponent: Infra
         tags: reconfigure
         testSteps:
-            1. Change number of CPU sockets and amount of memory while VM is runnng.
+            1. Add and remove the disk while VM is running
             2. Go to Compute -> infrastructure -> Virtual Machines -> Select Vm
             3. Go to VM reconfiguration
-            4.Change number of CPU sockets and amount of memory, save and submit
+            4. Click on Add Disk -> select disk_type and disk_mode , save and submit
             5. Check the count in VM details page
+            6. Remove the disk and Check the count in VM details page
     """
-    pass
+    orig_config = small_vm.configuration.copy()
```

Comment:
all these steps are exactly like in `test_vm_reconfig_add_remove_disk_cold`, please consider to avoid repeating the same code